ddtrace/contrib/sqlite3/patch.py

```diff
@@ -75,3 +76,13 @@ def __init__(self, conn, pin=None, cursor_cls=None):
     def execute(self, *args, **kwargs):
         # sqlite has a few extra sugar functions
         return self.cursor().execute(*args, **kwargs)
+
+    # backup was added in Python 3.7
+    if sys.version_info >= (3, 7, 0):
```

Comment:
makes more sense to do a `hasattr`?

Reply:
I almost did that.... but then decided since it stdlib and a known specific version added, that this was just as safe ?

I don't have a strong opinion, let me know if you think `hasattr` is better, I will update

Reply:
I don't, I think they both work.